app/controllers/posts_controller.rb

```diff
@@ -568,7 +569,7 @@ def unlock
   end
 
   def feature
-    return not_found(errors: ['no_privilege']) unless current_user.is_moderator
+    return not_found(errors: ['no_privilege']) unless current_user&.at_least_moderator?
```

Comment:
Untested suggestion:

```suggestion
    return not_found(errors: ['no_privilege']) if current_user&.standard?
```

(This one seems very much personal taste - suggesting in case it's of interest.)